setup.py

```diff
@@ -7,6 +7,6 @@
     name="kshape",
     version=__version__,
     description="Python implementation of kshape",
-    packages=find_packages(),
+    packages=find_packages(exclude=["*.tests"]),
```

Comment:
I would actually keep them. In some Linux distros they run tests to check if a package is still working.

Reply:
For sure, but if you keep the tests then you are putting a package named `tests.` into your list of packages, which means it can be used by people who are installing the software. This isn't what we usually want, because people just want to run the kshape code, and not the tests.

Devs, on the other hand like to run tests, and you can still get them, and run them, if you clone the git repo.

Reply:
I implemented now the same scheme as https://github.com/dateutil/dateutil
It allows to run tests on the installed package itself without polluting the namespace with top-level `tests` module

Reply:
Oh I see, thanks for pointing that out.